src/pages/en/guides/integrations.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Integrating with Astro
+---
+
+Members of the Astro community have been successfully integrating several third-party services into their Astro websites!
```

Comment:
maybe worth pulling this page out of this PR? I have some thoughts about the best way to get this out with community involvement, so that we can start collecting community content around different existing integrations

Reply:
Re: getting the page to work, I'm not sure what changes to the docs repo happened in the last 1 -2 weeks since I built my test page, but two things that I couldn't replicate in "real" docs that I had working were:

1. No background shading in the table that's holding the CodeSandbox/Stackblitz launch buttons. (Background colour should match the body, so the buttons appear just floating)

2. I had a nice script, with orange toggle, between npm and yarn init instructions. So, I'll need someone to reproduce that in the actual docs page, because my <style> and <script> don't seem to be working there.

Otherwise, the front page is good to make the swap, whenever you're ready!

Reply:
> Can do, but FYI the reason I included it was so that I had a link to point to from the main content. If this page isn't there (which is fine!) then that link on the main page will need to be removed, if not that section. (Maybe "coming soon", which I'm not as happy with.)

My main issue is that it feels empty, and "in progress". How about replacing these placeholder lists with a single, bulletted list of community content, with a call-to-action for community members to add their own community content. This would be good if you could seed it with a few links that you already know, either from Discord or our Twitter RTs.

Reply:
No problem, will be done in the morning! I've already been collecting several items.  Call to action is a great idea.

Reply:
Removed from this PR. Have another branch I will PR separately.